File: 107598019/HW2/Software Architecture_MySQL/adapter/presenter/EditCoursePresenter.cs
using Software_Architecture.usecase.editCourse;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Software_Architecture.adapter.presenter
{
    public class EditCoursePresenter : EditCourseOutput
    {
        public EditCoursePresenter() { }

        private string newTitle;

        public string getNewTitle()
        {
            return newTitle;
        }

        public void setNewTitle(string newTitle)
        {
            this.newTitle = newTitle;
        }
    }
}

File: 107598034/HW2/CourseSystem/src/Adapter/Presenter/EditCoursePresenter.java
package Adapter.Presenter;

import CourseUseCase.EditCourseUseCase.EditCourseOutput;

public class EditCoursePresenter implements EditCourseOutput {
	
	private String courseId;

	@Override
	public String getCourseId() {
		return courseId;
	}

	@Override
	public void setCourseId(String courseId) {
		this.courseId = courseId;
	}

}
